plugins.txt

```diff
@@ -49,12 +49,11 @@ extended-choice-parameter:0.78
 extended-read-permission:3.1
 external-monitor-job:1.7
 favorite:2.3.2
-fortify-on-demand-uploader:5.0.1
```

Comment:
null pointer when saving the global configuration page, breaks the slack plugin validation and isn't compatible will the configuration-as-code plugin,

I'm not comfortable with this plugin remaining on our jenkins instance